.github/workflows/scripts/eslint_disable_check.py

```
@@ -0,0 +1,159 @@
+            content = file.read()
+            return bool(eslint_disable_pattern.search(content))
+    except FileNotFoundError:
+        print(f"File not found: {file_path}")
+    except PermissionError:
+        print(f"Permission denied: {file_path}")
+    except (IOError, OSError) as e:
+        print(f"Error reading file {file_path}: {e}")
+    return False
+
+
+def check_eslint(files_or_directories):
```

Ruff (0.12.2)

Missing return type annotation for public function `check_eslint`

(ANN201)